src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
```

Comment:
Only `box-sizing` property allowed for `*` selector. `font-family` better move to body or html tag